x-pack/plugin/deprecation/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/deprecation/DeprecationHttpIT.java

```diff
@@ -59,7 +59,7 @@
 public class DeprecationHttpIT extends ESRestTestCase {
 
     /**
-     * Same as <code>DeprecationIndexingAppender#DEPRECATION_MESSAGES_DATA_STREAM</code>, but that class isn't visible from here.
+     * Same as <code>DeprecationIndexingAppender#CRITICAL_MESSAGES_DATA_STREAM</code>, but that class isn't visible from here.
```

Comment:
Is this an accidental change?

Reply:
ah yes.. looks like accidentally intellij refactored this for me too..